auth-web/src/stores/org.ts

```diff
@@ -699,6 +699,13 @@ export const useOrgStore = defineStore('org', () => {
     return result
   }
 
+  async function getStatementsSummary () {
+    const response = await PaymentService.getStatementsSummary(state.currentOrganization.id)
```

Comment:
Probably should have a try catch, that way the rest of the page keeps working even if the call fails. Here's an example where call fails and causes other code to have issues:

https://github.com/bcgov/sbc-auth/pull/2577